src/components/cap-case.js

```diff
@@ -11,6 +11,307 @@ export default class CapCase extends LitElement {
 		case: { type: String },
 	};
 
+	static styles = [
+		css`
+			/**/
+			/* Styles for HTML under our control */
+			/**/
+
+			/* .case-container */
+
+			.case-container {
+				flex: 0 0 83.33333%;
+				max-width: 83.33333%;
+				margin: auto;
+
+				padding-top: 0;
+				padding-bottom: 100px;
+
+				/*	styles inherited from "body" in capstone */
+
+				letter-spacing: 0.01em;
+				font-size: 20px;
+				font-weight: normal;
+				line-height: 29px;
+				color: var(--color-gray-500);
+				-moz-osx-font-smoothing: grayscale;
+				-webkit-font-smoothing: antialiased;
+				-webkit-hyphens: auto;
+				-moz-hyphens: auto;
+				-ms-hyphens: auto;
+				hyphens: auto;
+			}
+
+			/* .case-header */
+
+			.case-header {
+				font-size: 0.9em;
+				font-family: "Libre Baskerville", "Baskerville", serif;
+				text-align: center;
+				padding: 2em 2em 0;
+			}
+
+			@media (min-width: 992px) {
+				.case-header {
+					padding: 0;
+				}
+
+				.case-header::after {
+					content: '*';
+					color: var(--color-gray-000);
+				}
+			}
+
+			.case-header > h1 {
+				font-family: "Libre Baskerville", "Baskerville", serif;
+				font-weight: 700;
+				font-size: 1.3em;
+				line-height: 1.4em;
+				padding: 0;
+				margin: 0;
+			}
```

Comment:
More as an FYI rather than actual todo!

Following BEM, developers generally add a class to every block or element that they need to style, and then style that single class rather than a class + element. So here, instead of `casebody p` someone might name a css class `casebody__text`.

If there needs to be a stylistic variation of an existing block or element, BEM handles that with a modifier class. 

For example, 

```
<img class="image image--fullWidth" />

.image {
...
}

.image--fullWidth {
...
}
```